bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/Utils.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.easee.internal;
+
+import static org.openhab.binding.easee.internal.EaseeBindingConstants.*;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.easee.internal.model.ConfigurationException;
+import org.openhab.core.thing.Channel;
+import org.openhab.core.thing.type.ChannelTypeUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonNull;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+
+/**
+ * some helper methods.
+ *
+ * @author Alexander Friese - initial contribution
+ */
+@NonNullByDefault
+public final class Utils {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Utils.class);
+
+    /**
+     * only static methods no instance needed
+     */
+    private Utils() {
+    }
+
+    /**
+     * parses a date string in easee format to ZonedDateTime which is used by Openhab.
+     *
+     * @param date
+     * @return
+     */
+    public static ZonedDateTime parseDate(String date) throws DateTimeParseException {
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssX");
+        LOGGER.debug("parsing: {}", date);
+        ZonedDateTime zdt = ZonedDateTime.parse(date, formatter);
+        LOGGER.debug("parsing completed: {}", date);
```

Comment:
This could be reduced to TRACE level, if really needed at all?

Reply:
removed it.